pyproject.toml

```diff
@@ -22,10 +22,11 @@ dependencies = [
     'photutils >=1.6.0',
     'pyparsing >=2.4.7',
     'requests >=2.22',
-    # 'rad >=0.15.0',
-    'rad @ git+https://github.com/spacetelescope/rad.git@main',
-    # 'roman_datamodels >=0.15.0',
-    'roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git@main',
+    'rad == 0.17.0',
+#    'rad >= 0.16.0, < 0.17.1',
+#    'rad @ git+https://github.com/spacetelescope/rad.git@main',
+    'roman_datamodels ==0.17.0',
+#    'roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git@main',
```

Comment:
There absolutely should not be exact pins for these dependencies. This will cause us major headaches in development.